pycroft/model/finance.py

```diff
@@ -325,6 +325,25 @@ class BankAccountActivity(IntegerIdModel):
     )
 
 
+class MT940Error(IntegerIdModel):
+    mt940 = Column(Text(), nullable=False)
+    exception = Column(Text(), nullable=False)
+    author = relationship("User")
+    author_id = Column(Integer, ForeignKey("user.id"), nullable=False)
+    imported_at = Column(DateTimeTz, nullable=False,
+                       server_default=func.current_timestamp(),
+                       onupdate=func.current_timestamp())
+    bank_account = relationship(BankAccount, backref=backref("mt940_errors"))
+    bank_account_id = Column(Integer, ForeignKey(BankAccount.id),
+                             nullable=False)
+
+    def __init__(self, mt940, exception, author, bank_account):
```

Comment:
I don't think it is necessary to explicitly define the constructor.

Reply:
If the constructor is not defined explicitly, every attribute has to be set individually when creating a new entry.